sympy/polys/polytools.py

```diff
@@ -6218,8 +6216,44 @@ def sqf_list(f, *gens, **args):
     (2, [(x + 1, 2), (x + 2, 3)])
 
     """
-    return _generic_factor_list(f, gens, args, method='sqf')
-
+    options.allowed_flags(args, ['polys'])
+    opt = options.build_options(gens, args)
+    e = together(f)
+    gen_set = set()
+    coeff = 1
+    facs = []
+    for fac in Mul.make_args(e):
+        b, e = (fac.base, fac.exp) if fac.is_Pow else (fac, S.One)
+        _opt = opt.clone(dict(expand=True))
```

Comment:
This clone may not be necessary. It seems that `.expand()` is run anyway unless `opt.expand` is False. https://github.com/sympy/sympy/blob/6cd0b9fade27be198ee24987be0eac709413d297/sympy/polys/polyutils.py#L354-L357

Reply:
where can I find the definition of opt.clone() function?